### PR TITLE
Use template data for page tools if available

### LINKED ISSUE #435: Visual Editor incompatability, missing icons for page actions
## Describe the Bug
On every single page, there are no icons for the page actions. From what I can tell it's because selectors `#ca-history` are missing so the CSS doesn't apply. Because `#ca-edit` is missing, the skin is also no longer compatible with Visual Editor.

1. Go to [https://projectsekai.miraheze.org/wiki/Main_Page](https://projectsekai.miraheze.org/wiki/Main_Page)
2. Visit any page
3. Icons for Edit, Edit Source, History, etc. buttons are missing, and Visual Editor fails to work

## System
_Please complete the following information:_
 - MediaWiki version: 1.37.1 (2d847ba)
 - Server OS: Debian 11
 - Installed extensions and versions: [See here](https://projectsekai.miraheze.org/wiki/Special:Version)
 - PHP version: 7.4.28 (fpm-fcgi)

### PR DESCRIPTION
**Pull request by lens0021**
Fixes #435

| MediaWiki Version | Logged-out | Logged-in |
|--|--|--|
| 1.35.6 (Star Citizen Wiki uses 1.35.5) | image | image
| 1.39.0-alpha (Used by Wikipedia) | image | image |
| 1.37.1 (Used by Miraheze etc) | image | image

Thank you!

### CODE REVIEW

File: includes/Partials/PageTools.php [hunk 1]
```diff
@@ -65,17 +67,27 @@ public function buildPageTools(): array {
...
+				$actionshtml = $this->skin->getMenuData( 'actions', $contentNavigation[ 'actions' ] ?? [] );
+				$namespaceshtml = $this->skin->getMenuData( 'namespaces', $contentNavigation[ 'namespaces' ] ?? [] );
+				$variantshtml = $this->skin->getMenuData( 'variants', $contentNavigation[ 'variants' ] ?? [] );
+				$toolboxhtml = $this->skin->getMenuData( 'tb',  $portals['TOOLBOX'] ?? [] );
+			} else {
+				$viewshtml = $parentData['data-portlets']['data-views'];
+				$actionshtml = $parentData['data-portlets']['data-actions'];
+				$namespaceshtml = $parentData['data-portlets']['data-namespaces'];
+				$variantshtml = $parentData['data-portlets']['data-variants'];
+				$toolboxhtml = $parentData['data-portlets-sidebar']['array-portlets-rest'];
```
Comment (lens0021) [inline includes/Partials/PageTools.php]: Not sure about the toolbox; test needed

Comment (lens0021) [reply includes/Partials/PageTools.php]: Done

**Comment by alistair3149** [comment]
Thank you so much and I really appreciate the PR.
I want to apologize that due to some personal reason I was unable to keep up with MW development, but I'll be more active in the next few months to do a refactor for the next LTS.

**Comment by lens0021** [comment]
This PR is ready. +1

You don't have to apologize. I'm completely okay and happy to hear about the upcoming refactoring. :)

**Pull request merged**